lib/mix/tasks/phoenix.gen.html.ex

```diff
@@ -39,6 +39,8 @@ defmodule Mix.Tasks.Phoenix.Gen.Html do
     binding = binding ++ [plural: plural, route: route,
                           inputs: inputs(attrs), params: Mix.Phoenix.params(attrs)]
 
+    Mix.Phoenix.check_module_name_availability!(binding[:module])
```

Comment:
This should be called twice:

     Mix.Phoenix.check_module_name_availability!(binding[:module] <> "Controller")
     Mix.Phoenix.check_module_name_availability!(binding[:module] <> "View")